server/profileFormSubmission.js

```diff
@@ -0,0 +1,35 @@
+const { Pool } = require('pg');
+const { allowedNodeEnvironmentFlags } = require('process');
+const pool = new Pool({ 
```

Comment:
We really should create global configs considering this is going to be used for every endpoint lol, I'll bring it up at our next retro.